### PR TITLE
Tests index show

### PR DESCRIPTION
**Pull request by MaaooVaass**
## Summary
text for model book

## Card 
https://trello.com/c/z381r4Zx/21-crear-tests-para-el-modelo-de-book

### CODE REVIEW

**Comment by fedeest** [comment]
All good. Fix conflicts are remember to merge previous PR first and then change this PR base.

File: spec/controllers/books_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe BooksController do
+  describe 'GET #index' do
+    subject(:http_request) do
```
Comment (lmasello) [inline spec/controllers/books_controller_spec.rb]: There may be many http requests. Use a more descriptive name, such as `index_request`. Also, you can do this in just one line with curly brackets (`{}`).

File: spec/controllers/books_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      get :index
+    end
+
+    let!(:books) { create_list(:book, 70) }
+
+    it 'responses with the books json' do
```
Comment (lmasello) [inline spec/controllers/books_controller_spec.rb]: typo. responds/returns?

File: spec/controllers/books_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      expect(JSON.parse(http_request.body)['total_count']).to eq(Book.all.count)
+    end
+
+    it 'responds with 200 status' do
+      expect(http_request).to have_http_status(:ok)
+    end
+  end
+
+  describe 'GET #show' do
+    subject(:http_request) do
```
Comment (lmasello) [inline spec/controllers/books_controller_spec.rb]: According to the previous comment. Here you can use `show_request`.

**Review by lmasello — CHANGES_REQUESTED** [review]

**Review by lmasello — APPROVED** [review]

**Pull request merged**